### PR TITLE
Profile proofs display tweaks

### PR DESCRIPTION
**Pull request by chromakode**
:eyeglasses: @keybase/react-hackers 

A small precursor to DESKTOP-1422. Noticed some discrepancies vs. the mockups and wanted to discuss them in a separate PR. I added a `metaRevoked` constant since we didn't seem to have a way to display the "REVOKED" badge as pictured in mockups.

### CODE REVIEW

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits d8e38aa1b972...406e1652a9a7 introduced some visual changes on linux:
 * Added: **Profile-Your_Profile_Broken**  
   Profile-Your_Profile_Broken rendered
 * Added: **Profile-Revoked_Scrolled**  
   Profile-Revoked_Scrolled rendered
 * Added: **Profile-Revoked**  
   Profile-Revoked rendered
 * Changed: **Search_User_Pane-Unfollowed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search_User_Pane-Unfollowed-d8e38aa1b972.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search_User_Pane-Unfollowed-406e1652a9a7.png)  
   Search_User_Pane-Unfollowed comparison
 * Changed: **Search_User_Pane-Followed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search_User_Pane-Followed-d8e38aa1b972.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search_User_Pane-Followed-406e1652a9a7.png)  
   Search_User_Pane-Followed comparison
 * Changed: **Search_User_Pane-Changed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search_User_Pane-Changed-d8e38aa1b972.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search_User_Pane-Changed-406e1652a9a7.png)  
   Search_User_Pane-Changed comparison
 * Changed: **Search_User_Pane-Broken_tracker_not_following** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search_User_Pane-Broken_tracker_not_following-d8e38aa1b972.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search_User_Pane-Broken_tracker_not_following-406e1652a9a7.png)  
   Search_User_Pane-Broken_tracker_not_following comparison
 * Changed: **Search-Waiting** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search-Waiting-d8e38aa1b972.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search-Waiting-406e1652a9a7.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search-Waiting.png)
 * Changed: **Search-Searching** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search-Searching-d8e38aa1b972.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search-Searching-406e1652a9a7.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search-Searching.png)
 * Changed: **Search-Group** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search-Group-d8e38aa1b972.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search-Group-406e1652a9a7.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search-Group.png)
 * Changed: **Search-Chat_enabled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search-Chat_enabled-d8e38aa1b972.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search-Chat_enabled-406e1652a9a7.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Search-Chat_enabled.png)
 * Changed: **Profile-Your_Profile_empty** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Your_Profile_empty-d8e38aa1b972.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Your_Profile_empty-406e1652a9a7.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Your_Profile_empty.png)
 * Changed: **Profile-Your_Profile** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Your_Profile-d8e38aa1b972.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Your_Profile-406e1652a9a7.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Your_Profile.png)
 * Changed: **Profile-Unfollowed_Scrolled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Unfollowed_Scrolled-d8e38aa1b972.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Unfollowed_Scrolled-406e1652a9a7.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Unfollowed_Scrolled.png)
 * Changed: **Profile-Unfollowed_Profile_page** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Unfollowed_Profile_page-d8e38aa1b972.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Unfollowed_Profile_page-406e1652a9a7.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Unfollowed_Profile_page.png)
 * Changed: **Profile-Unfollowed_Folders_Expanded** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Unfollowed_Folders_Expanded-d8e38aa1b972.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Unfollowed_Folders_Expanded-406e1652a9a7.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Unfollowed_Folders_Expanded.png)
 * Changed: **Profile-Unfollowed_Few_Folders** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Unfollowed_Few_Folders-d8e38aa1b972.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Unfollowed_Few_Folders-406e1652a9a7.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Unfollowed_Few_Folders.png)
 * Changed: **Profile-Unfollowed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Unfollowed-d8e38aa1b972.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Unfollowed-406e1652a9a7.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Unfollowed.png)
 * Changed: **Profile-Followed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Followed-d8e38aa1b972.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Followed-406e1652a9a7.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Followed.png)
 * Removed: **Profile-Changed_Scrolled** [(view original)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Changed_Scrolled-d8e38aa1b972.png)
 * Removed: **Profile-Changed** [(view original)](https://keybase-app-visdiff.s3.amazonaws.com/1469230524628-d8e38aa1b972-406e1652a9a7-linux/Profile-Changed-d8e38aa1b972.png)

File: shared/common-adapters/user-proofs.shared.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ export function metaColor (proof: Proof): string {
     case metaUnreachable: color = globalColors.red; break
     case metaPending: color = globalColors.black_40; break
     case metaDeleted: color = globalColors.red; break
+    case metaRevoked: color = globalColors.red; break
```
Comment (chrisnojima) [inline shared/common-adapters/user-proofs.shared.js]: i'm confused. i think metaDeleted is revoked

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 025627c29da5...ba974c7d25d2 introduced some visual changes on linux:
 * Added: **Profile-Your_Profile_Broken**  
   Profile-Your_Profile_Broken rendered
 * Changed: **Search_User_Pane-Unfollowed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search_User_Pane-Unfollowed-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search_User_Pane-Unfollowed-ba974c7d25d2.png)  
   Search_User_Pane-Unfollowed comparison
 * Changed: **Search_User_Pane-Followed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search_User_Pane-Followed-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search_User_Pane-Followed-ba974c7d25d2.png)  
   Search_User_Pane-Followed comparison
 * Changed: **Search_User_Pane-Changed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search_User_Pane-Changed-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search_User_Pane-Changed-ba974c7d25d2.png)  
   Search_User_Pane-Changed comparison
 * Changed: **Search_User_Pane-Broken_tracker_not_following** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search_User_Pane-Broken_tracker_not_following-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search_User_Pane-Broken_tracker_not_following-ba974c7d25d2.png)  
   Search_User_Pane-Broken_tracker_not_following comparison
 * Changed: **Search-Waiting** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search-Waiting-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search-Waiting-ba974c7d25d2.png)  
   Search-Waiting comparison
 * Changed: **Search-Searching** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search-Searching-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search-Searching-ba974c7d25d2.png)  
   Search-Searching comparison
 * Changed: **Search-Group** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search-Group-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search-Group-ba974c7d25d2.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search-Group.png)
 * Changed: **Search-Chat_enabled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search-Chat_enabled-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search-Chat_enabled-ba974c7d25d2.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Search-Chat_enabled.png)
 * Changed: **Profile-Your_Profile_empty** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Your_Profile_empty-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Your_Profile_empty-ba974c7d25d2.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Your_Profile_empty.png)
 * Changed: **Profile-Your_Profile** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Your_Profile-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Your_Profile-ba974c7d25d2.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Your_Profile.png)
 * Changed: **Profile-Unfollowed_Scrolled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Unfollowed_Scrolled-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Unfollowed_Scrolled-ba974c7d25d2.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Unfollowed_Scrolled.png)
 * Changed: **Profile-Unfollowed_Profile_page** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Unfollowed_Profile_page-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Unfollowed_Profile_page-ba974c7d25d2.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Unfollowed_Profile_page.png)
 * Changed: **Profile-Unfollowed_Folders_Expanded** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Unfollowed_Folders_Expanded-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Unfollowed_Folders_Expanded-ba974c7d25d2.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Unfollowed_Folders_Expanded.png)
 * Changed: **Profile-Unfollowed_Few_Folders** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Unfollowed_Few_Folders-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Unfollowed_Few_Folders-ba974c7d25d2.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Unfollowed_Few_Folders.png)
 * Changed: **Profile-Unfollowed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Unfollowed-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Unfollowed-ba974c7d25d2.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Unfollowed.png)
 * Changed: **Profile-Followed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Followed-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Followed-ba974c7d25d2.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Followed.png)
 * Changed: **Profile-Changed_Scrolled** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Changed_Scrolled-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Changed_Scrolled-ba974c7d25d2.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Changed_Scrolled.png)
 * Changed: **Profile-Changed** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Changed-025627c29da5.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Changed-ba974c7d25d2.png) [(diff)](https://keybase-app-visdiff.s3.amazonaws.com/1469483505213-025627c29da5-ba974c7d25d2-linux/Profile-Changed.png)

**Comment by chromakode** [comment]
@chrisnojima Ok removed all the `metaRevoke` stuff from this.

**Comment by chrisnojima** [comment]
👍

**Pull request merged**